conda_env/cli/main.py

```diff
@@ -1,47 +1,34 @@
 # Copyright (C) 2012 Anaconda, Inc
 # SPDX-License-Identifier: BSD-3-Clause
-"""Entry point for all conda-env subcommands."""
+"""DEPRECATED: Use `conda.cli.main_env` instead.
```

Comment:
This `DEPRECATED` line, will show up in the API docs, IIRC. It's a good-enough work-around, until we have a better overview of all deprecated modules and objects.